package.json

```diff
@@ -130,7 +130,7 @@
 		"test-server:coverage": "npm run -s test-server -- --coverage",
 		"test-server:watch": "npm run -s test-server -- --watch",
 		"translate": "i18n-calypso --format pot --output-file ./calypso-strings.pot -k translate,__,_x,_n,_nx -e date '**/*.js' '**/*.jsx' '**/*.ts' '**/*.tsx' '!build/**' '!packages/**/dist/**' '!public/**' '!**/test/**' '!node_modules/**' '!apps/full-site-editing/**' '!**/node_modules/**'",
-		"typecheck": "tsc --noEmit",
+		"typecheck": "NODE_OPTIONS='--max-old-space-size=4096' tsc --project client",
```

Comment:
This is used in the `typecheck-strict` check:

https://github.com/Automattic/wp-calypso/blob/b2a3ab3444bc3f19721970a403f90fe5f2fdf008/.circleci/config.yml#L246

which thus now reads, expanded:

```
> NODE_OPTIONS='--max-old-space-size=4096' tsc --project client "--project" "client/landing/gutenboarding"
``` 

So we probably add an dedicated `script` to package.json and use it in the CCI config, or solve the problem there.